test/addon_selector_test.rb

```diff
@@ -66,7 +66,8 @@
 
       it "does not display any popup" do
         # stub empty Yast::Popup so any method call would raise an exception
-        stub_const("Yast::Popup", double)
```

Comment:
Does that mean that any such `stub_const("Yast::Foo", double)` will fail or has bug? We have more

https://github.com/search?q=org%3Ayast+stub_const&type=code

Reply:
yast/yast-update/test/inst_update_partition_auto_test.rb
15:      stub_const("Yast::FileSystems", double)
yast/yast-installation/test/snapshots_finish_test.rb
10:    stub_const("Yast::StorageSnapper", double)